backend/sozisel/test/sozisel_web/schema/events/quiz_mutations_test.exs

```diff
@@ -0,0 +1,267 @@
+defmodule SoziselWeb.Schema.QuizMutationsTest do
+  use SoziselWeb.AbsintheCase
+
+  import Sozisel.Factory
+
+  alias Sozisel.Model.Events
+
+  @create_quiz """
+  mutation CreateQuiz($input: CreateQuizInput!) {
+    createQuiz(input: $input) {
+      id
+      name
+      start_minute
+      event_structure {
+        ... on Quiz {
+          duration_time_sec
+          target_percentage_of_participants
+          tracking_mode
+          quiz_questions {
+              question
+              answers
+              correct_answers
+          }
+        }
+      }
+      session_template_id
+    }
+  }
+  """
+
+  @update_quiz """
+  mutation UpdateQuiz($input: UpdateQuizInput!) {
+    updateQuiz(input: $input) {
+      id
+      name
+      start_minute
+      event_structure {
```

Comment:
those things that you return from graphql query should all be in cammel case not in snake case. Absinthe maps all these types and when you write queries just like in this place it is in the same form as frontend will call the backend.